src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ballFirst = lottery.getRandomBall();
+        System.out.println(ballFirst);
+        Ball ballSecond = lottery.getRandomBall();
+        System.out.println(ballSecond);
+        Ball ballThird = lottery.getRandomBall();
+        System.out.println(ballThird);
     }
```

Comment:
Instead of creating multiple instances of `Ball` and printing them one by one, consider using a loop to simplify your code. This will adhere to the DRY principle and make your code more maintainable.